internal/config/docs.toml

```diff
@@ -0,0 +1,624 @@
+[WebServer.TLS]
+# The location of the TLS certificate file.
+CertPath = '/home/$USER/.chainlink/tls/server.crt' # Example
+# The hostname configured for TLS to be used by the Chainlink node. This is useful if you configured a domain name specific for your Chainlink node.
+Host = 'tls-host' # Example
+# The location of the TLS private key file.
+KeyPath = '/home/$USER/.chainlink/tls/server.key' # Example
+# The port used for HTTPS connections. Set this to `0` to disable HTTPS. Disabling HTTPS also relieves Chainlink nodes of the requirement for a TLS certificate.
+HTTPSPort = 6689 # Default
+# Forces TLS redirect for unencrypted connections.
+ForceRedirect = false # Default
+
+[JobPipeline]
+# Defines the maximum size for HTTP requests and responses made by `http` and `bridge` adapters.
+HTTPRequestMaxSize = '32768' # Default
+# Defines the default timeout for HTTP requests made by `http` and `bridge` adapters.
+DefaultHTTPRequestTimeout = '15s' # Default
+# Enables the External Initiator feature. If disabled, `webhook` jobs can ONLY be initiated by a logged-in user. If enabled, `webhook` jobs can be initiated by a whitelisted external initiator.
+ExternalInitiatorsEnabled = false # Default
+# The maximum time allowed for a single job run. If it takes longer, it will exit early and be marked errored. If set to zero, disables the time limit completely.
+MaxRunDuration = '10m' # Default
+# In order to keep database size manageable, Chainlink nodes will run a reaper that deletes completed job runs older than a certain threshold age. `ReaperInterval` controls how often the job pipeline reaper will run.
+#
+# Set to `0` to disable the periodic reaper.
+ReaperInterval = '1h' # Default
+# Determines the age limit for job runs. Completed job runs older than this will be automatically purged from the database.
+ReaperThreshold = '24h' # Default
+# Some jobs write their results asynchronously for performance reasons such as OCR. `JOB_PIPELINE_RESULT_WRITE_QUEUE_DEPTH` controls how many writes will be buffered before subsequent writes are dropped.
+#
+# Do not change this setting unless you know what you are doing.
+ResultWriteQueueDepth = 100 # Default
+
+[FluxMonitor]
+DefaultTransactionQueueDepth = 1 # Default
+# Enable transaction simulation for Flux Monitor.
+SimulateTransactions = false # Default
+
+[OCR2]
+ContractConfirmations = 3 # Default
+BlockchainTimeout = '20s' # Default
+ContractPollInterval = '1m' # Default
+ContractSubscribeInterval = '2m' # Default
+ContractTransmitterTransmitTimeout = '10s' # Default
+DatabaseTimeout = '10s' # Default
+# TODO
+KeyBundleID = '7a5f66bbe6594259325bf2b4f5b1a9c900000000000000000000000000000000' # Example
+# TODO
+MonitoringEndpoint = 'test-mon-end' # Example
+
+# This section applies only if you are running off-chain reporting jobs.
+[OCR]
+ObservationTimeout = '11s'
+BlockchainTimeout = '3s'
+ContractPollInterval = '1h'
+ContractSubscribeInterval = '1m'
```

Comment:
Note that these correspond directly to internal OCR configuration variables. See: https://github.com/smartcontractkit/libocr/blob/9e4afd8896f365b964bdf769ca28f373a3fb0300/offchainreporting2/types/local_config.go#L9

It probably makes sense to just link to the OCR docs for each of these.

I wonder if we could even be clever enough to pull the exact SHA of libocr we use for the generated docs and include a link to it?

Reply:
This script lives in the module so it would already import the same libocr version - it might actually be easy to just reference the type and field in a standard way that allows to extract the text from the comment in libocr. Then we don't have to link to something unfamiliar looking.
I could see an argument for keeping these separate though (even if they are very similar), e.g. if the godoc is more dev oriented than nodeop oriented, or if we want to make use of the additional styling features from markdown that don't work in the godoc.